mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/models/LegStep.java

```diff
@@ -78,4 +84,36 @@ public StepManeuver getManeuver() {
   public List<StepIntersection> getIntersections() {
     return intersections;
   }
+
+
+  /**
+   * An optional string indicating the name of the rotary.
+   *
+   * @return String with the rotary name
+   * @since 2.0.0
+   */
+  public String getRotaryName() {
+    return rotaryName;
+  }
+
+  /**
+   * An optional string indicating the pronunciation of the name of the rotary.
+   *
+   * @return String in IPA with the rotary name's pronunciation
+   * @since 2.0.0
+   */
+  public String getRotaryPronunciation() {
+    return rotaryPronunciation;
+  }
+
+  /**
+   * The pronunciation hint of the way name. Will be undefined if no pronunciation is hit.
+   *
+   * @return String with the pronunciation
+   * @since 2.0.0
+   */
+  public String getPronunciation() {
```

Comment:
@ghoshkaj why two getters for the same field `getRotaryPronunciation` and `getPronunciation`?